### DISCUSSION TITLE
Feature Request: Please add RTL support

### DISCUSSION THREAD

**Discussion by mohammad-reza-rahbar** [ueberdosis/tiptap#5440] [Feature Requests]
### Description

Please add RTL support.

### Use Case

A lot of multilingual websites need to support Right-to-left scripts like Arabic, Persian and Hebrew.

### Type

New feature

**Comment by bdbch** [comment]
I'm not very familiar with RTL languages so I'd need some more information.

What exactly is an issue with Tiptap right now with RTL languages (or rather what would need to be improved?)

**Reply by nperez0111** [reply]
From my understanding rtl is supported already by prosemirror & therefore tiptap, but you need to use the CSS. To support bi-directional text (where you can set the text direction per node) we may want to take an approach like remirror: https://remirror.io/docs/api/extension-bidi/

**Reply by mohammad-reza-rahbar** [reply]
The problem is, I cannot change the [direction ](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) of each node, either manually by setting `dir="rtl"`/`dir="ltr"` or automatically detected by `dir="auto"`.

**Reply by nperez0111** [reply]
I believe with global attributes you can add the `dir` attribute to each node you need and allow it to be set to anything you like: https://tiptap.dev/docs/editor/extensions/custom-extensions/extend-existing#global-attributes
